cli/tests/testdata/npm/cjs_yargs/main.js

```diff
@@ -0,0 +1,20 @@
+import yargs from "npm:yargs@15.4.1";
+
+const args = yargs(["serve", "8000"])
+  .command("serve [port]", "start the server", (yargs) => {
+    return yargs
+      .positional("port", {
+        describe: "port to bind on",
+        default: 5000,
+      });
+  }, (argv) => {
+    console.info(`start server on :${argv.port}`);
+  })
+  .option("verbose", {
+    alias: "v",
+    type: "boolean",
+    description: "Run with verbose logging",
+  })
+  .argv;
+
+console.log(args);
```

Comment:
Is this needed since it looks like it gets ignored in the output?

Reply:
Something went wrong during a merge, this test was included in https://github.com/denoland/deno/pull/16547. And yes, it's required, this call failed before the fix